rpcs3/main.cpp

```diff
@@ -130,8 +130,10 @@ LOG_CHANNEL(q_debug, "QDEBUG");
 		dlg.exec();
 	};
 
-#ifdef __APPLE__
+#if defined(__APPLE__) && defined(BLOCKS)
 	// Cocoa access is not allowed outside of the main thread
+	// Prevents crash dialogs from freezing the program
+	// BLOCKS is required for dispatch_sync, but GCC-11 does not support it
```

Comment:
nitpick, but can you move this comment to the define above (also the one for the include)
`#if defined(__APPLE__) && defined(BLOCKS) // BLOCKS is required for dispatch_sync, but GCC-11 does not support it`

Reply:
Done